app/main.py

```diff
@@ -1,3 +1,5 @@
+from app.characteristics_of_a_knight.knight import Knights
+from app.Battle.battle_knight import BattleKnights
 KNIGHTS = {
```

Comment:
Move `KNIGHTS` variable to a separate file `knights_config.py` to not overload `main.py` file